Does not compile with Mbed OS master

`[ERROR] Library name 'littlefs' is not unique (defined in '/builds/ws/myapp/myapp/storage-selector/littlefs/mbed_lib.json' and '/builds/ws/myapp/myapp/mbed-os/features/filesystem/littlefs/mbed_lib.json')
`

Broken with this commit: 

https://github.com/ARMmbed/mbed-os/commit/2e1c2a1cdf168dd739e777165b21f6cd0509c188

--- a/littlefs.lib
+++ b/littlefs.lib
@@ -1,1 +1,0 @@
-https://github.com/ARMmbed/mbed-littlefs/#72fab82453c60668a7b9b2c0b17f3482398d926a

